src/index.js

```diff
@@ -171,7 +171,7 @@ export type RequestInfo = {|
   /**
    * The result of executing the operation.
    */
-  result: ExecutionResult | null,
+  result: ExecutionResult,
```

Comment:
Removed the null here because `result` could never be null (see [here](https://github.com/graphql/express-graphql/blob/master/src/index.js#L330))

Reply:
Thanks for comment 👍 
It's very helpful.